### DISCUSSION TITLE
html file?

### DISCUSSION THREAD

**Discussion by Reznar705** [yancharkin/SpelunkyClassicHD#25] [General]
is there an html file i can download to run the game? i can't find out how to download it on my chromebook and i know there's an html website where i can play it but i was wondering how i can download it if not that's ok

**Comment by yancharkin** [comment]
In the releases section of the html5 version repo you can download '...electron_ready.zip' file.
https://github.com/yancharkin/SpelunkyClassicHDhtml5/releases/tag/1.1.7-french
It is html version almost without additional files.
But it's unlikely that it'll work without webserver or electron or something similar.
I have never used ChromeOS, but as far as I know it is possible to run Linux apps on it (in developer mode?). Maybe it's a better option.
